movies/movies/TableView/AboutMovieViewController.swift

```diff
@@ -47,17 +48,21 @@ final class AboutMovieViewController: UIViewController {
 
     func fetchDataMovie(cell: AboutMovieTableViewCell, indexPath: IndexPath, idMovie: Int) {
         presenter.fetchDataAboutMovie(idMovie: idMovie)
-        cell.aboutMoviesLabel.text = presenter.aboutMovie?.overview
-        guard let release = presenter.aboutMovie?.releaseDate else { return }
+        cell.aboutMoviesLabel.text = presenter.newMovie?.overview
+        guard let release = presenter.newMovie?.releaseDate else { return }
         cell.releaseDateLabel.text = "Даты выхода: \(release))"
-        guard let mark = presenter.aboutMovie?.voteAverage else { return }
+        guard let mark = presenter.newMovie?.voteAverage else { return }
         cell.voteLabel.text = "Оценка \(mark)"
-        cell.titleMoviesLabel.text = presenter.aboutMovie?.title
+        cell.titleMoviesLabel.text = presenter.newMovie?.title
     }
 
     func fetchImage(cell: AboutMovieTableViewCell) {
```

Comment:
```suggestion
    func fetchImage(cell: AboutMovieTableViewCell) {
```
перенеси внутрь ячейки